tests/e2e/util_e2e/utility_simple.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import typing
+import mgclient
+import sys
+import pytest
+
+
+def test_does_throw_if_null_is_checked_against_in_generic_case():
```

Comment:
I would like to see all the cases from the issue covered here https://github.com/memgraph/memgraph/issues/319

Reply:
Actually I told to Gábor to add these cases to `tests/gql_behave/tests/memgraph_V1/features/case.feature`. This test suite is meant to test the semantics of case.